### PR TITLE
LPS-96737 Remove usages of alloyui in select_layout_page_template_entry.jsp

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/add_asset_selector.jsp [hunk 1]
```diff
@@ -105,17 +105,15 @@ String redirect = PortalUtil.getLayoutFullURL(layout, themeDisplay);
...
+		const selectedOption = assetEntryTypeSelector.options[
+			assetEntryTypeSelector.selectedIndex
+		];
 
-		var index = assetEntryTypeSelector.get('selectedIndex');
-
-		var selectedOption = assetEntryTypeSelector.get('options').item(index);
-
-		var url = selectedOption.attr('data-url');
+		const url = selectedOption.getAttribute('data-url');
```
Comment (jbalsas) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/add_asset_selector.jsp]: I think our conversation is leading to using `dataset`, right?

File: modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/configuration.jsp [hunk 2]
```diff
@@ -54,13 +54,14 @@ String redirect = ParamUtil.getString(request, "redirect");
 	</liferay-frontend:edit-form-footer>
 </liferay-frontend:edit-form>
 
-<aui:script sandbox="<%= true %>" use="aui-base">
-	var form = A.one('#<portlet:namespace />fm');
+<aui:script require="metal-dom/src/all/dom as dom">
+	const assetEntryIdInput = document.getElementById('<portlet:namespace />assetEntryId');
+	const articlePreview = document.getElementById('<portlet:namespace />articlePreview');
```
Comment (jbalsas) [inline modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/configuration.jsp]: Sort?

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/select_layout_page_template_entry.jsp [hunk 3]
```diff
@@ -183,15 +187,14 @@ renderResponse.setTitle(LanguageUtil.get(request, "select-template"));
 					},
 					id: '<portlet:namespace />addLayoutDialog',
 					title: '<liferay-ui:message key="add-page" />',
-					uri: actionElement.getData('add-layout-url')
+					uri: actionElement.getAttribute('data-add-layout-url')
```
Comment (jbalsas) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/select_layout_page_template_entry.jsp]: `dataset`?

File: modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/edit_site_navigation_menu.jsp [hunk 4]
```diff
@@ -321,28 +278,58 @@ sb.append("/js/SiteNavigationMenuItemDOMHandler.es as siteNavigationMenuItemDOMH
...
-				var sidebarHeaderButton = A.one('#<portlet:namespace />sidebarHeaderButton');
+		const formData = new FormData();
+
+		Object.keys(data).forEach(
+			function(key) {
+				formData.append('<portlet:namespace />' + key, data[key]);
+			}
+		);
+
+		fetch(
```
Comment (jbalsas) [inline modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/edit_site_navigation_menu.jsp]: Use `Liferay.Util.fetch`

File: modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/edit_site_navigation_menu.jsp [hunk 5]
```diff
@@ -321,28 +278,58 @@ sb.append("/js/SiteNavigationMenuItemDOMHandler.es as siteNavigationMenuItemDOMH
...
 
-	var setSidebarBody = function(content) {
-		AUI().use(
-			['aui-base', 'aui-parse-content'],
-			function(A) {
-				var sidebarBody = A.one('#<portlet:namespace />sidebarBody');
-				var sidebarHeaderButton = A.one('#<portlet:namespace />sidebarHeaderButton');
+		const formData = new FormData();
+
+		Object.keys(data).forEach(
```
Comment (jbalsas) [inline modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/edit_site_navigation_menu.jsp]: Can you check `objectToFormData` in `frontend-js-web` and see if it's equivalente so we can use it?

Also, please file a guidelines issue to track this down.

**Comment by p2kmgcl** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/74410

**Pull request closed without merging**